backend/api/Areas/Notification/Controllers/TemplateController.cs

```diff
@@ -131,16 +136,19 @@ public IActionResult DeleteNotificationTemplate([FromBody] NotificationTemplateM
         /// </summary>
         /// <param name="templateId"></param>
         /// <param name="to"></param>
-        /// <param name="model"></param>
+        /// <param name="projectId"></param>
         /// <returns></returns>
-        [HttpPost("{templateId}")]
+        [HttpPost("{templateId}/projects/{projectId}")]
         [HasPermission(Permissions.SystemAdmin)]
         [Produces("application/json")]
         [ProducesResponseType(typeof(Models.Queue.NotificationQueueModel), 201)]
         [ProducesResponseType(typeof(ErrorResponseModel), 400)]
         [SwaggerOperation(Tags = new[] { "notification" })]
-        public async Task<IActionResult> SendNotificationAsync(int templateId, string to, [FromBody] object model = null) // TODO: Provide a way to pass a model.
+        public async Task<IActionResult> SendProjectNotificationAsync(int templateId, string to, int projectId)
```

Comment:
Provides a way to manually test any project notification.  It is always sent to the user account you are logged in as.